man/systemd-nspawn.xml

```diff
@@ -1082,6 +1097,12 @@
       <programlisting># chcon system_u:object_r:svirt_sandbox_file_t:s0:c0,c1 -R /srv/container
 # systemd-nspawn -L system_u:object_r:svirt_sandbox_file_t:s0:c0,c1 -Z system_u:system_r:svirt_lxc_net_t:s0:c0,c1 -D /srv/container /bin/sh</programlisting>
     </example>
+
+    <example>
+      <title>Run a container with an OSTree deployment</title>
+
+      <programlisting># systemd-nspawn -b -i ~/image.raw --pivot-root=/ostree/deploy/os/deploy/checksum:/sysroot --bind=+/sysroot/ostree/deploy/os/var:/var</programlisting>
```

Comment:
btw, if i got this right, then "checksum" is supposed to be an actual checksum, i.e. a variable? Maybe write this as `$CHECKSUM` in order to clarify this?